tests/ArrayCache/ArrayCacheDecoratorExtraTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Yiisoft\Cache\Tests\ArrayCache;
 
+require_once __DIR__ . '/../MockHelper.php';
```

Comment:
Why it is not handled handled by autoloader?

Reply:
Previously it had different namespace but in this PR I’ve changed it and I think now it can be autoloaded, I’ll check.